db/migrate/20150714140850_remove_new_from_compute_attributes.rb

```diff
@@ -0,0 +1,35 @@
+class FakeComputeAttribute < ActiveRecord::Base
+  set_table_name 'compute_attributes'
+  serialize :vm_attrs, Hash
+end
+
+class RemoveNewFromComputeAttributes < ActiveRecord::Migration
+  def up
+    FakeComputeAttribute.all.each do |comp_attr|
+      attrs_name = ['nics', 'interfaces'].find do |attrs_name|
```

Comment:
This should handle the two `interfaces_attrs_name` values we have depending on the CR.